test/integration/dialects/sqlite/issue-11862.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
File name is missing `.test.`, I don't think this test is even running

Reply:
:man_facepalming: